src/collective/cover/layout.py

```diff
@@ -233,6 +242,112 @@ def update(self):
                 i += 1
 
 
+# Ok we exclude a little too much here. We exclude "-89", which is invalid, but
+# we also exclude "-abc" which IS a valid class, but it's also a silly CSS
+# class, so why bother to support it!?
+VALID_CSS_RE = re.compile('[a-zA-Z_][0-9a-zA-Z_-]*')
+
+def cssConstraint(value):
```

Comment:
Follow PEP8 in the name of the function https://www.python.org/dev/peps/pep-0008/#function-names